tests/regression_tests/automl/utils.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+
+from ludwig.datasets import dataset_registry
+from ludwig.datasets.base_dataset import BaseDataset
+
+REGISTRY = {"adult_census_income", "mnist"}
```

Comment:
nit: rename to `TEST_DATASET_REGISTRY` and add a comment about what this registry is used for, i.e. "Subset of Ludwig Zoo datasets use for automl type inference regression tests."

Reply:
agree, there are very many registries in the codebase so this helps dismbiguate ;)